resources/js/Pages/Collections/Components/CollectionsVoteReceivedModal/CollectionsVoteReceivedModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { useTranslation } from "react-i18next";
+import { ButtonLink } from "@/Components/Buttons/ButtonLink";
+import { Dialog } from "@/Components/Dialog";
+
+export const CollectionsVoteReceivedModal = ({
+    onClose,
+    collection,
+}: {
+    onClose: () => void;
+    collection?: {
```

Comment:
@shahin-hq, my intention is to display the modal when the collection is passed; otherwise, I need to pass both a show value and a collection (refer to the logic in the "show vote modal" button).

Consider that I don't see this approach as the final one. It might depend on how we display the modal once everything is merged together. It can be updated later according to that. What do you think?

Reply:
Ah, now I got, sure, we can keep this as is, we can refine it in future if we have to. Thanks for the explanation!